service/src/main/kotlin/no/nav/su/se/bakover/service/avstemming/AvstemmingsPeriode.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.su.se.bakover.service.avstemming
+
+import java.time.Instant
+
+data class AvstemmingsPeriode(
+    val fom: Instant,
+    val tom: Instant
```

Comment:
Jeg syns det er litt dumt at vi kaller ting `fom` og `tom`; de er visuelt så veldig like. Kunne vi ha gått for `fraOgMed` og `tilOgMed` i stedet?

Reply:
@ingara fikser du en pr og tar det for hele prosjektet da eller? ;)

Reply:
Hehe, høres ut som en fin jobb for en rolig dag :)